worlds/yoshisisland/__init__.py

```diff
@@ -0,0 +1,410 @@
+import os
+import typing
+import threading
+import dataclasses
+
+from typing import Dict, List, Set, TextIO
+from BaseClasses import Item, MultiWorld, Location, Tutorial, ItemClassification
+from worlds.AutoWorld import World, WebWorld
+from Options import PerGameCommonOptions
+import Patch
```

Comment:
```suggestion

from typing import List, Set, TextIO
from BaseClasses import Item, MultiWorld, Location, Tutorial, ItemClassification
from worlds.AutoWorld import World, WebWorld
```

Remove unused imports